data/cpu_bugs/autoyast/SLE-12-SP4/autoyast_gnome.xml

```diff
@@ -0,0 +1,161 @@
+      <listentry>
+        <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+        <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+        <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+        <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+        <accept_verification_failed config:type="boolean">true</accept_verification_failed>
+        <ask_on_error config:type="boolean">false</ask_on_error>
+        <import_gpg_key config:type="boolean">true</import_gpg_key>
+        <media_url>http://download.suse.de/ibs/SUSE/Updates/SLE-SERVER/12-SP4/x86_64/update/</media_url>
+        <product>update</product>
+        <product_dir>/</product_dir>
+      </listentry>
+  </add-on>
+  <deploy_image>
+    <image_installation config:type="boolean">false</image_installation>
+  </deploy_image>
+  <general>
+    <ask-list config:type="list"/>
+    <mode>
+      <halt config:type="boolean">true</halt>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">true</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">false</accept_verification_failed>
+      <import_gpg_key config:type="boolean">true</import_gpg_key>
+    </signature-handling>
+    <storage>
+      <partition_alignment config:type="symbol">align_optimal</partition_alignment>
+      <start_multipath config:type="boolean">false</start_multipath>
+    </storage>
+  </general>
+  <keyboard>
+    <keyboard_values>
+      <delay/>
+      <discaps config:type="boolean">false</discaps>
+      <numlock>bios</numlock>
+      <rate/>
+    </keyboard_values>
+    <keymap>english-us</keymap>
+  </keyboard>
+  <language>
+    <language>en_US</language>
+    <languages/>
+  </language>
+  <login_settings/>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">0</timeout>
+    </errors>
+    <messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">10</timeout>
+    </messages>
+    <warnings>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">10</timeout>
+    </warnings>
+    <yesno_messages>
+      <log config:type="boolean">true</log>
+      <show config:type="boolean">true</show>
+      <timeout config:type="integer">10</timeout>
+    </yesno_messages>
+  </report>
```

Comment:
commonly the autoyast profiles used for openQA tests define no timeout on messages to catch everything uncovered.

Reply:
No, I need it timeout and automatic enter system. because this autoyast not for "installation testing", it just for helping me to make an "object" that could be migrated between two hosts. My monitor script need this VM automatic start up and print "login prompt", so that , monitor script could know the VM is working, and migrating testing could start.